core/block.js

```diff
@@ -337,13 +337,11 @@ Blockly.Block.prototype.unplug = function(opt_healStack) {
 
 /**
  * Returns all connections originating from this block.
- * @param {boolean} all If true, return all connections even hidden ones.
+ * @param {boolean} _all If true, return all connections even hidden ones.
  * @return {!Array.<!Blockly.Connection>} Array of connections.
  * @private
  */
-Blockly.Block.prototype.getConnections_ = function(
-    /* eslint-disable no-unused-vars */ all
-    /* eslint-enable no-unused-vars */) {
+Blockly.Block.prototype.getConnections_ = function(_all) {
```

Comment:
The duplications in code between this and BlockSvg.getConnections_ is mildly worrying. Is there a way we can get the BlockSvg to call back to the Block.prototype.getConnections(..) after checking rendered state?

Since this is tangental to this PR, it could be a TODO and future follow up.